src/main/java/core/basesyntax/AreaAndFullInfo.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax;
+
+public interface AreaAndFullInfo {
```

Comment:
bad interface name, sounds more like variable name rather interface name
```suggestion
public interface AreaCalculator {
```